src/components/Message.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Message(props) {
+  const message = props.message;
```

Comment:
Also, here it's better practice when we use object destructuring : const {message} = props;